SDL_ttf.c

```diff
@@ -3128,6 +3128,44 @@ int TTF_SetFontScriptName(TTF_Font *font, const char *script)
 #endif
 }
 
+extern DECLSPEC int SDLCALL TTF_GetScript(Uint32 ch, char* script)
```

Comment:
This function should probably take a `scriptlen` parameter and document that it should be at least 5 characters long and then null terminate the string.

Reply:
I don't think so. The script is always 4 characters long. I've added a check to cover that just like it happens in TTF_SetFontScriptName.

Reply:
actually that always 4 char, so maybe "char script[4]" (or 5 to have it null terminated)

Reply:
@1bsyl TTF_SetFontScriptName does exactly the same thing... why not keep it consistent?

Reply:
yes this is right ... that was just a suggestion.